controllers/v1/observationsController.js

```diff
@@ -527,7 +532,7 @@ module.exports = class Observations extends Abstract {
    * @returns {JSON} List of entities in observations.
    */
 
-  async searchEntities(req) {
+  async searchEntitiesOld(req) {
```

Comment:
@borkarsaish65 what is the requirement for this function name change?

Reply:
I just kept the old function in the codebase for any future referencing. New function uses entity management service.